src/locales.json

```diff
@@ -146,7 +147,8 @@
         "category": "Diese Seite wurde leider nicht gefunden.",
         "event": "Dieses Event wurde leider nicht gefunden.",
         "extra": "Dieses Extra wurde leider nicht gefunden.",
-        "disclaimer": "Impressum ist leider für diese Stadt nicht verfügbar."
+        "disclaimer": "Impressum ist leider für diese Stadt nicht verfügbar.",
```

Comment:
Why not "_Das_ Impressum" or "_Ein_ Impressum?

Reply:
Not my task, changed anyway though.